lib/internal/crypto/util.js

```diff
@@ -308,6 +309,17 @@ function bigIntArrayToUnsignedInt(input) {
   return result;
 }
 
+function bigIntArrayToBigInt(input) {
+  let result = 0n;
+
+  for (let n = 0; n < input.length; ++n) {
+    const n_reversed = input.length - n - 1;
+    result |= BigInt(input[n]) << 8n * BigInt(n_reversed);
+  }
+
+  return result;
+}
```

Comment:
Suggestion, feel free to ignore: Out of curiosity, did you try benchmarking this versus

```js
BigInt(`0x${Buffer.from(input).toString('hex')}`)
```

Or, if performance really is a concern in this code path,

```js
BigInt(`0x${Buffer.from(input.buffer, input.byteOffset, input.byteLength).toString('hex')}`)
```

(I know that this is essentially the same as the existing function `bigIntArrayToUnsignedInt` above, I am just curious what the performance impact is.)